### ISSUE TITLE
Problem with spacing between buttons 

### ISSUE THREAD

**Issue by andresossa13** [andresossa13/final-assignment-andresossa13#2]
Helo dear Goradz, my favourite teacher ever. Im having problems separating some butttons  that I did in my landing page (index.html),
They are in a flex column but I can't find the way to reduce the space between them, if you preview my page you can see thay are very separated from each other. 

Please hellppp!

Thanks

**Comment by gorazd** [comment]
Hello, apologies for responding only now.

I downloaded your work, and if i'm not mistaken you solved this by centering them?

**Comment by andresossa13** [comment]
Yes sir. I'm currently in the proccess of adding a google font and I will be just missing the mediaquery. Would you help me upload it to my current domain on tuesday class please?

**Comment by gorazd** [comment]
it would be my pleasure!
